### DISCUSSION TITLE
Auto fetch on start up

### DISCUSSION THREAD

**Discussion by TrongHieu90** [gitextensions/gitextensions#9677] [Q&A] [answered]
Hi, I tried to search but could not find how to make git extensions auto fetch on startup, or fetch after a certain period of time (like Source Tree)

Is there any way to set this up ?

**Comment by RussKie** [comment, marked as answer]
There's a periodic background fetch plugin. Check the Settings dialog to
configure it.

**Reply by TrongHieu90** [reply]
Nice i found it, is there also a way to force fetch on first startup ?

**Comment by RussKie** [comment]
Don't believe so. But you can click Fetch button.

On Oct 24, 2021 8:39 PM, "Trong Hieu Nguyen" ***@***.***>
wrote:

Nice i found it, is there also a way to force fetch on first startup ?

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/gitextensions/gitextensions/discussions/9677#discussioncomment-1527083>,
or unsubscribe
